src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -8245,6 +8245,11 @@ void ExportMusicXml::writeMeasureTracks(const Measure* const m,
                 const track_idx_t endtrack = staff2track(spannerStaff + 1);
                 spannerStop(this, starttrack, endtrack, seg->tick(), partRelStaffNo, spannersStopped);
 
+                // We check if there are also dynamic annotations
+                if (seg->findAnnotation(ElementType::DYNAMIC, strack, etrack - 1)) {
```

Comment:
Looks good - I have one suggestion. Since https://github.com/musescore/MuseScore/pull/25741, any text based item can be added to a timetick segment.  This should extend the functionality to all of these items as well.
```suggestion
                for (EngravingItem* annotation : seg->annotations()) {
                    if (annotation->track() != track || !annotation->isTextBase()) {
                        continue;
                    }
```

Reply:
I agree.

I made the minimum change to stick only to the request.

Reply:
@miiizen , this proposed change has made the Visual Tests fail. I don't know why. Do you have any idea about the problem ?

```
Run echo "This PR appears to change some of the visual tests."
This PR appears to change some of the visual tests.
Please carefully review the new visual test results in the uploaded artifact that can be found here: https://github.com/musescore/MuseScore/actions/runs/13282708891
Error: Process completed with exit code 1.
```

Reply:
A rebase will fix those!